dist/css/styles-minimal.dev.css

```diff
@@ -405,27 +405,132 @@ template {
 [hidden] {
   display: none; }
```

Comment:
Are we all happy with the pattern of having the closing curly brace on the last line? Is there a reason for this other than just a stylistic choice?

Reply:
No, that's wrong - I didn't lint the code! I'll add that task to the travis tasks.

Reply:
I'm so very glad to hear that because I hate it so much ;)

Reply:
Bad news. It has nothing to do with the lint task - it's the compile task. The lint enforces the source to be good, but the compile has its own ideas. I'm going to ignore this for the time being, but it's something that we should revisit.